wiki/HelpWanted.md

```diff
@@ -16,8 +16,6 @@ At the moment, the greatest need is for beta testers, but you can also contribut
 
 ### Programming tasks
 
-  * Windows: make and maintain an official installer.
-
   * macOS: build and distribute each release.
```

Comment:
Is this still relavent? We have the CD handle builds now, and we've been able to handle distribution for a while now.